### PR TITLE
engine/schema: Use semantically correct upgrade path 4.6.2->4.7.0

### PR DESCRIPTION
**Pull request by bhaisaab**
Adds an explicit upgrade path class from 4.6.2 to 4.7.0, that extends over
the upgrade path from 4.6.1 to 4.7.0

cc @remibergsma @DaanHoogland -- from #1247

### CODE REVIEW

**Comment by remibergsma** [comment]
@bhaisaab Thanks, makes sense. Will test an upgrade and report back.

**Comment by bhaisaab** [comment]
Ping @remibergsma

**Comment by wido** [comment]
Did the tests succeed?

**Comment by remibergsma** [comment]
Didn't have a chance to test it yet.

**Comment by DaanHoogland** [comment]
I don't see the semantic necessity for this. doe upgrades work different if  Upgrade461to470() is used insted of an empty Upgrade462to470()? It is just adding a code duplication.

**Comment by remibergsma** [comment]
For the record: the upgrade works:

```
2016-01-23 16:06:44,796 INFO  [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) DB version = 4.6.2 Code Version = 4.7.1-SNAPSHOT
2016-01-23 16:06:44,797 INFO  [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) Database upgrade must be performed from 4.6.2 to 4.7.1-SNAPSHOT
2016-01-23 16:06:44,797 DEBUG [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) Running upgrade Upgrade462to470 to upgrade from 4.6.2-4.7.0 to 4.7.0
2016-01-23 16:06:44,800 DEBUG [c.c.u.s.Script] (main:null) (logid:) Looking for db/schema-461to470.sql in the classpath
```

And then:
```
2016-01-23 16:06:44,803 DEBUG [c.c.u.d.ScriptRunner] (main:null) (logid:) -- Schema upgrade from 4.6.1 to 4.7.0;
```

**Comment by bhaisaab** [comment]
I've heard everyone's comments on this, I stand by my PR. The argument that DB upgrade works is not relevant to what this PR fixes.

**Comment by remibergsma** [comment]
@bhaisaab Let me rephrase: I tested your PR and that works :-)

**Comment by DaanHoogland** [comment]
Ok let me rephrase as well: this is extra code without extra functionality: :-1:

**Comment by bhaisaab** [comment]
Such a waste
